File: Jessica/70.Climbing Stairs.cpp
class Solution {
public:
    int climbStairs(int n) { 
        if(n==1)
            return 1;
        if(n==2)
            return 2;
        int ret=0;
        int StepOne=1;
        int StepTwo=2;
        for(int i=2;i<n;i++){
            ret = StepOne+StepTwo;
            StepOne=StepTwo;
            StepTwo=ret;
                
            
        }
        return ret;
    }
};

File: Jessica/70.Climbing Stairs.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n==1:
            return 1
        if n==2:
            return 2
        StepOne=1
        StepTwo=2
        ret=0
        for i in range(3,n+1):
            ret=StepOne+StepTwo
            StepOne=StepTwo
            StepTwo=ret
        return ret
            
        